frontend/components/vault/Vault.vue

```diff
@@ -0,0 +1,278 @@
+import AnimatedArrow from '../common/other/AnimatedArrow.vue';
+import TextBlock from '~/components/common/other/TextBlock.vue';
+import Button from '~/components/common/inputs/BaseButton.vue';
+import FormatAddress from '~/components/common/formatters/FormatAddress.vue';
+import FormatCurrency from '~/components/common/formatters/FormatCurrency.vue';
+import Loading from '~/components/common/other/Loading.vue';
+import AnimatedNumber from '~/components/common/formatters/AnimatedNumber.vue';
+
+export default Vue.extend({
+    name: 'Vault',
+    components: {
+        AnimatedArrow,
+        TimeTill,
+        Loading,
+        FormatCurrency,
+        TextBlock,
+        Button,
+        FormatAddress,
+        AnimatedNumber,
+        Alert,
+        Tooltip,
+    },
+    props: {
+        vaultTransaction: {
+            type: Object as Vue.PropType<VaultTransaction>,
+            default: null,
+        },
+        vaultId: {
+            type: String,
+            required: true,
+        },
+        error: {
+            type: String,
+            default: '',
+        },
+        isExplanationsShown: {
+            type: Boolean,
+            default: true,
+        },
+        areVaultsFetching: {
+            type: Boolean,
+            default: false,
+        },
+        network: {
+            type: String,
+            default: 'mainnet',
+        },
+    },
+    data() {
+        return {
+            isTableExpanded: false,
+        };
+    },
+    computed: {
+        vaultError(): { error: string; showBanner: boolean } | null {
+            if (this.error) {
+                return {
+                    error: this.error,
+                    showBanner: true,
+                };
+            } else if (!this.areVaultsFetching && !this.vaultTransaction) {
+                return {
+                    error: 'This vault was not found',
+                    showBanner: true,
+                };
+            } else if (!this.areVaultsFetching && this.vaultTransaction.state === 'liquidated') {
+                return {
+                    error: 'This vault has already been liquidated',
+                    showBanner: false,
+                };
+            }
+            return null;
+        },
+        vaultLink(): string | undefined {
+            if (!this.vaultTransaction || !this.vaultTransaction.auctionId) {
+                return undefined;
+            }
+            const link = generateLink(this.network, 'collateral');
+            const formattedAuctionId = this.vaultTransaction.auctionId.replace(':', '%3A');
+
+            return `${link}&auction=${formattedAuctionId}`;
+        },
+    },
+    watch: {
+        isExplanationsShown: {
+            immediate: true,
+            handler(newIsExplanationsShown) {
+                if (!newIsExplanationsShown) {
+                    this.isTableExpanded = true;
+                }
+            },
+        },
+    },
```

Comment:
Is this used?

Reply:
This again is from the other Auction components:

https://github.com/sidestream-tech/unified-auctions-ui/blob/main/frontend/components/auction/collateral/CollateralAuction.vue#L388

We do use the functionality here to expand the table, so I assume it is needed, if we need it in the other ones.